pom.xml

```diff
@@ -684,9 +684,9 @@ under the License.
 
 			<!-- commons collections needs to be pinned to this critical security fix version -->
 			<dependency>
-				<groupId>commons-collections</groupId>
```

Comment:
@MartijnVisser hi, Martijn. If we can not fix it in flink-connector-kafka, can we solved it in flink here? not replace it, but add a dependency. when flink-connector-kafka update, check and remove it in the future?